src/modules/webRTC/components/SidebarButton.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from "../../components/Box";
+import { Button } from "../../components/Button";
+import { T } from "../../components/Text";
+import { sidebarButtonProps, textProps } from "./styles";
+
+export type SidebarPages = "chat" | "user" | "offerings" | "answerings"
+
+export const SidebarButton = ({height, text, refPage, page, setPage}: {height: number, text: string, refPage: SidebarPages,  page: SidebarPages, setPage: React.Dispatch<SidebarPages>}) => {
+
+    return (
+        <Box>
+            <Button 
+                zIndex={1}
```

Comment:
also explicitly setting zIndexes might lead to issues down the line, it's much better to rely on layouts unless absolutely necessary